modules/olm-catalogsource.adoc

```diff
@@ -75,8 +83,11 @@ Set the `olm.catalogImageTemplate` annotation to your index image name and use o
 * `configmap`: OLM parses config map data and runs a pod that can serve the gRPC API over it.
 --
 <8> Specify the value of `legacy` or `restricted`. If the field is not set, the default value is `legacy`. In a future {product-title} release, it is planned that the default value will be `restricted`. If your catalog cannot run with `restricted` permissions, it is recommended that you manually set this field to `legacy`.
-<9> Automatically check for new versions at a given interval to stay up-to-date.
-<10> Last observed state of the catalog connection. For example:
+<9> Optional: For source type `grpc` with an `image` reference, overrides the default node selector for the pod serving the content in `spec.image`.
```

Comment:
It's meant to mean the following, like in the intro to the other topic:
> When an Operator Lifecycle Manager (OLM) catalog source of source type grpc defines a spec.image

Was hoping to save space here but I knew it would be tricky. 😅 I'll just go with the longer-form, hopefully more-clear version.